src/Notifications/NotificationCenterController.cs

```diff
@@ -83,6 +83,7 @@ internal NotificationCenterController(DynamoView view, DynamoLogger dynLogger)
             dynamoView.LocationChanged += DynamoView_LocationChanged;
             notificationsButton.Click += NotificationsButton_Click;
             dynamoView.Closing += View_Closing;
+            dynamoView.PreviewMouseDown += DynamoView_PreviewMouseDown;
```

Comment:
Please also unsubscribe this event handler somewhere, you can refer to where we call `dynamoView.Closing -= View_Closing`

Reply:
@QilongTang  done!